src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -328,6 +337,20 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
                         isMinSelectableChoicesEnabled ? Integer.toString(minSelectableChoices) : "-1");
     }
 
+    /**
+     * Get the number of MSQ choices when ticking the option "students (excluding self)".
```

Comment:
Don't mention `when ticking the option "students (excluding self)"`; how the method is being used is independent of what the method does.

Reply:
Just to be sure, so when writing Javadoc comments we only write what the method does?

Reply:
Yes. You can optimize a method for its caller, but ultimately a method is its own construct and independent of its callers.